highfidelity/hifi
# Case 21046: Fix Subsurface Scattering and ResourceCache "extra" bug

https://highfidelity.manuscript.com/f/cases/21046/Fix-ResourceCache-extra-functionality-SSS

Test plan:
- Run [this](https://gist.githubusercontent.com/SamGondelman/57f015697165c575f632e0b9489c7e54/raw/4cd9674fe9a2dc2ec8217ef4d931787c0be3f850/ScatteringAndExtraTest.js).  You'll see:
![image](https://user-images.githubusercontent.com/7650116/52362956-90d3bd00-29f6-11e9-9139-4c8f62aef269.png)
- Open luci and select the "Scattering" debug framebuffer.  Use your middle mouse button to make the debug region bigger.  You'll see something like this:
![image](https://user-images.githubusercontent.com/7650116/52363074-c9739680-29f6-11e9-9d1f-5f6441b10ea2.png)
- Wear the Matthew avatar.  With luci still open, you should see something like this:
![image](https://user-images.githubusercontent.com/7650116/52363155-f162fa00-29f6-11e9-9d18-23773a82c618.png)
- Go to some domains with lots of models and textures.  Make sure things load as in master.
- Verify that skyboxes load.
- Verify that mirrors and the HMD preview work.
